src/main/bash/sdkman-main.sh

```diff
@@ -155,8 +152,20 @@ function sdk() {
 			"$native_command" "$QUALIFIER"
 		elif [ -z "$4" ]; then
```

Comment:
I think you can even reduce the rest of the `elif` chain to

```
elif [[ $# < 10 ]]; then
  "$native_command" "$QUALIFIER" "${@:3}"
else
  ___sdkman_help
fi
```

`${@:3}` is basically a slice from element 3 to the end.

Reply:
Good shout @helpermethod, don't know why I didn't think of that myself :stuck_out_tongue_closed_eyes: 

I've even gone a step further and removed the whole `if` `elif` chain to make it even simpler:

```
"$native_command" "${@:2}"
```

Triggering help at the end isn't a requirement, so I'm just propagating all the parameters now.